### ISSUE TITLE
Errors evaluating a CNN model (Keras)

### ISSUE THREAD

**Issue by krishnakundan1980** [microsoft/Windows-Machine-Learning#352]
I'm submitting a bug encountered, didn't find exactly the same issue reported.

## Current behavior

I built a Keras CNN (LeNet 5 architecture) which has standard 5 layers of following operations:
    tf.keras.layers.Conv2D(6, kernel_size=5, strides=1,  activation='tanh', input_shape=(256,256,3), padding='same'), #C1
    tf.keras.layers.AveragePooling2D(), #S2
    tf.keras.layers.Conv2D(16, kernel_size=5, strides=1, activation='tanh', padding='valid'), #C3
    tf.keras.layers.AveragePooling2D(), #S4
    tf.keras.layers.Flatten(), #Flatten
    tf.keras.layers.Dense(120, activation='tanh'), #C5
    tf.keras.layers.Dense(84, activation='tanh'), #F6
    tf.keras.layers.Dense(3, activation='softmax')]) #Output layer

Then, I converted it into ONNX format using Keras2onnx. I can load the converted ONNX model and run for inferring task successfully using onnxruntime. I used Python 3.8.3, Tensorflow 2.3.0 and Keras 2.4.3 for creating my CNN model.

Now, I am using the converted ONNX model on another window 10 machine using the [steps](https://docs.microsoft.com/en-us/windows/ai/windows-ml/get-started-desktop). I can successfully run it for the provided sample SqueezeNet.onnx model. But when I replace with my Keras trained CNN model + making other changes such as my model name, labels etc. I get exception exactly at this line of code shown below in the image.

![image](https://user-images.githubusercontent.com/17733649/94090929-c4ca4480-fdcb-11ea-998e-88865895f21a.png)
![image](https://user-images.githubusercontent.com/17733649/94090960-dd3a5f00-fdcb-11ea-9b6f-141446717fe8.png)

I am providing correct input layer name, "conv2d_input" also image loaded correctly because essentially I using the sample code which runs fine for SqueezeNet.onnx model.

The attached model version (keras2onnx 1.7.0, onnx 1.7.0, opset_version 7, tf2onnx 1.6.3 ) runs without errors using the onnxruntime 1.4.0 python package.

Is Windows ML not capable of running such models?

## Expected behavior

Inference task should be successful as it is working in onnxruntime with python.

## Minimal reproduction of the problem with instructions
Please see above behavior section.

## Environment
OS Version (Windows 10 Enterprise):
Visual Studio 
- [ ] 2017 (version: 15.9.27 professional)
- CppWinRT: 2.0.200729.8
- WinMLTools version 1.5.2

**Comment by zhangxiang1993** [comment]
Hi @krishnakundan1980, Thanks for reporting this.
We are looking into this now. Can you share the model with us? I didn't see the attached model.

**Comment by zhangxiang1993** [comment]
There is a chance that the nominal pixel range is not supported in WinML if your model is trained not with normalized_0_255 pixel value. In that case, you will need to modify your model and download the latest WinML nuget package or manually tensorize your videoframe by following our public sample. 
If so, I can provide more details. Please let me know.

**Comment by krishnakundan1980** [comment]
Hi @zhangxiang1993, Thanks for looking into the problem. Please find attached herewith the model which I used. Please unzip to see .h5 model.
I think my model is trained with 0-255 normalized value only. Please let me know.

**Comment by krishnakundan1980** [comment]
My mode file size was large to be attached here, so am sharing through my google drive.
https://drive.google.com/drive/folders/1o0aYL4TQsCvgU5dTrA5RaXdVVhSNV0kB?usp=sharing

**Comment by zhangxiang1993** [comment]
The order of the input is NHWC, while binding a ImageFeatureValue, WinML would fail at attempting to bind image but input feature shape is invalid. 

Convert keras nhwc to onnx nchw format by using channel_first_inputs parameter as below,
`onnx_model = keras2onnx.convert_keras(model, channel_first_inputs=["conv2d_input"])`
